src/Stripe.net/Services/Sources/SourceService.cs

```diff
@@ -95,7 +95,7 @@ public virtual async Task<Source> DetachAsync(string customerId, string sourceId
 
         public virtual async Task<StripeList<Source>> ListAsync(string customerId, SourceListOptions listOptions = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I think we couldn't use `DeleteEntity` before because it returned a `StripeDeleted` object, but detaching a source always returned a source.

Not sure about `GetEntityList`. We can likely use it as well.